### ISSUE TITLE
0.2.10 still the latest version?

### ISSUE THREAD

**Issue by baixiongnull** [trumank/mint#194]
I noticed that many comments, including the author's reply, mentioned the new version of the fix, but the latest version of the release page is still stuck in the 0.2.10 version a year ago.
Did the author publish the latest version elsewhere?

**Comment by jieyouxu** [comment]
No, the changes on master is not ready yet for another release, they are still bugs and further testing that needs to be conducted. There is no concrete timeline for a next release at this moment.

**Comment by Mitch2025** [comment]
Software will never be released if you wait to only release it without bugs. Software will always have bugs. The current release is filled with them that are fixed in the current code. Can you mention what some of these show stopping bugs are in the current code that isn't present in the year old release? Current code has fixes for various mods and some major fixes for the MS Store version so mods can actually save their configs as well as additional features that's all missing from the current release. And now there are even things being fixed specifically for season 5 according to the latest commits. Can't you just release it and give a warning that it has bugs?
